doc/source/cluster/vms/getting-started.rst

```diff
@@ -29,43 +29,50 @@ Setup
 
 Before we start, you will need to install some Python dependencies as follows:
 
-.. tabs::
+.. tab-set::
 
-   .. tab:: Ray Team Supported
+   .. tab-item:: Ray Team Supported
+        :sync: Ray Team Supported
 
-      .. tabs::
+      .. tab-set::
 
-         .. tab:: AWS
+         .. tab-item:: AWS
+            :sync: AWS
 
             .. code-block:: shell
 
                 $ pip install -U "ray[default]" boto3
 
-         .. tab:: GCP
+         .. tab-item:: GCP
+            :sync: GCP
 
             .. code-block:: shell
 
                 $ pip install -U "ray[default]" google-api-python-client
 
-   .. tab:: Community Supported
+   .. tab-item:: Community Supported
+        :sync: Community Supported
```

Comment:
<img width="720" alt="Screenshot 2023-11-21 at 10 57 42 AM" src="https://github.com/ray-project/ray/assets/5459654/b66bd8d9-bc01-4b0a-82f7-e4f80090998b">
This part looks messed up. Can we revert the changes in this table?

Reply:
Specifically the fact that `:sync:` appears and the title of the tabs are duplicated in this table and similar tables on this page.

Reply:
Nice find. `:sync:` should synchronize the tab state of all the tabs on this page; there's probably a spacing issue or similar causing this. Will fix.